packages/api/src/models/BillPayments.ts

```diff
@@ -9,6 +9,23 @@ enum PaymentType {
 	check = 'Check',
 	credit = 'Credit',
 	cash = 'Cash',
+	bankTransfer = 'Bank Transfer',
+	creditCard = 'Credit Card',
+	debit = 'Debit',
+	payPal = 'PayPal',
+	twoCheckout = '2Checkout',
+	visa = 'VISA',
+	mastercard = 'MASTERCARD',
+	discover = 'DISCOVER',
+	nova = 'Nova',
+	amex = 'AMEX',
+	diners = 'DINERS',
+	eurocard = 'EUROCARD',
+	jcb = 'JCB',
+	ach = 'ACH',
+	googleCheckout = "Google Checkout",
+	amazon = "Amazon",
```

Comment:
Hmm. Sorry to pester again, but I think the amazon and google ones are internal only as they don't seem to be in the frontend list: https://github.com/freshbooks/magnum-ui/blob/main/app/utils/payment-types.js

Reply:
np :) I'remove them.